resources/public/index.html

```diff
@@ -32,17 +32,17 @@
         localStorage.setItem("day8.re-frame-10x.show-panel","\"true\"")
         localStorage.setItem("day8.re-frame-10x.using-trace?","\"true\"")
       }
-      function isBrowserSupported() {
-        return userAgent.indexOf("chrome/") > -1;
+      function isBrowserUnsupported() {
+        return userAgent.indexOf("applewebkit/") > -1;
```

Comment:
This isn't right, the user agent for chrome under macos is `'mozilla/5.0 (macintosh; intel mac os x 10_15_7) applewebkit/537.36 (khtml, like gecko) chrome/100.0.4896.75 safari/537.36'`

Reply:
Brave is also the above.

Firefox is `"mozilla/5.0 (macintosh; intel mac os x 10.15; rv:98.0) gecko/20100101 firefox/98.0"`.

Safari is `"mozilla/5.0 (macintosh; intel mac os x 10_15_7) applewebkit/605.1.15 (khtml, like gecko) version/15.4 safari/605.1.15"`

Reply:
I've updated this check and can verify it works as follows:

MacOS:
- Safari shows the `Safari based browsers based are not supported` message
- Chrome, Firefox and Brave don't

iOS:
- All browsers show the `Safari based browsers based are not supported` message.